reframe/frontend/cli.py

```diff
@@ -617,7 +617,8 @@ def main():
                     printer.info(runner.stats.performance_report())
 
         else:
-            printer.info('No action specified. Exiting...')
+            printer.info("No action specified. Please specify '-l' for "
+                         "listing or '-r' for running.")
```

Comment:
I would use this type of quoting: `` `-l' `` and, yes use both `-l` and `-L` in the description, as you proposed.